EAD/session 3/Assignment1/WebContent/myWebContent/searchStudent.jsp

```diff
@@ -0,0 +1,47 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Search Student</title>
+<style type="text/css">

```

Comment:
Do follow to include css and js in other files and in future all assignments

Reply:
Ok